apidoc/arcgis.geocoding.html

```diff
@@ -653,23 +653,30 @@
 <dl class="method">
 <dt id="arcgis.geocoding.geocode">
 <code class="descclassname">geocoding.</code><code class="descname">geocode</code><span class="sig-paren">(</span><em>address</em>, <em>search_extent=None</em>, <em>location=None</em>, <em>distance=None</em>, <em>out_sr=None</em>, <em>category=None</em>, <em>out_fields=’*’</em>, <em>max_locations=20</em>, <em>magic_key=None</em>, <em>for_storage=False</em>, <em>geocoder=None</em><span class="sig-paren">)</span><a class="headerlink" href="#arcgis.geocoding.geocode" title="Permalink to this definition">¶</a></dt>
-<dd><p>The geocode function geocodes one location per request.</p>
+<dd><p>The geocode function geocodes one location per request. This function corresponds to the findAddressCandidates operation at
```

Comment:
@jyaistMap Updates to the API doc needs to be made in the source code repo. The apidoc is generated by running sphinx on the source code and any changes made here will be overwritten. Can you remove the changes from here and send a pull request for these changes in the source code repo for geosaurus?